.eslintrc.js

```diff
@@ -125,20 +114,5 @@ module.exports = {
 				},
 			],
 		},
-		{
```

Comment:
Ich habe keine Ahnung ob das für irgendwas in der ci/tests gebraucht wird/wurde. :D

Reply:
Vllt. gab es früher unterschied. aber wir haben jetzt *.spec.ts einheitlich